RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc

```diff
@@ -359,7 +359,7 @@ int HGCalImagingAlgo::findAndAssignClusters(std::vector<KDNode> &nd,KDTree &lp,
       if(flag_isolated) nd[i].data.isBorder = true; //the hit is more than delta_c from any of its brethren
     }
     //check if this border hit has density larger than the current rho_b and update
-    if(nd[i].data.isBorder && rho_b[ci] < nd[i].data.rho)
+    if(nd[i].data.isBorder && rho_b[ci] <= nd[i].data.rho)
       rho_b[ci] = nd[i].data.rho;
```

Comment:
I'm somehow missing how an additional (incremental to what it was before)
``` if (rho_b[ci] == nd[i].data.rho) rho_b[ci] = nd[i].data.rho; ```
can change anything (note that rho_b is just a simple vector of doubles, so no possible difference  after full assignment if there is an equality)

Please clarify.